pkg/proxy/ipvs/proxier.go

```diff
@@ -339,25 +334,17 @@ func NewProxier(ipFamily v1.IPFamily,
 	healthzServer *healthcheck.ProxierHealthServer,
 	scheduler string,
 	nodePortAddressStrings []string,
-	kernelHandler KernelHandler,
+	kernelVersion *version.Version,
```

Comment:
So the theory here is that passing in the `kernelHandler` / `kernelVersion` as an argument means that this is more unit testable... except that we don't ever call `NewProxier` in any unit tests (they construct the proxier by hand), and we never use `kernelHandler`/`kernelVersion` _after_ `NewProxier`.

So you could just remove this as an argument and call `utilkernel.GetVersion()` directly from `NewProxier`.